patches/postcss-urlrebase+1.3.0.patch

```diff
@@ -0,0 +1,11 @@
+diff --git a/node_modules/postcss-urlrebase/index.d.ts b/node_modules/postcss-urlrebase/index.d.ts
```

Comment:
Why is this change required?

Reply:
Something about fixing the typescript types. But @sirreal would know more precisely.

Reply:
A browser package (editor, I believe) uses postcss in the browser.

This dependency expected to work in a node environment (most common for postcss stuff) and use the URL type imported from node. This is not available in the environment. However, it's fine to use the URL global, which is what this change does. Otherwise, the package would error when it tried to import a non-existent (node.js) type.

Reply:
https://github.com/strarsis/postcss-urlrebase/pull/2

Reply:
Add package patch note here: https://github.com/WordPress/gutenberg/pull/63015